### PR TITLE
85713 Increase maximum number of applicants per application - form 10-10d

### PR DESCRIPTION
**Pull request by michaelclement**
## Summary

This PR increases the limit on how many applicants a user may add to a single 10-10D application. This is a front-end only change, the backend functionality to accompany this is currently in progress.

This PR also includes a couple small unit tests and some cleanup, not directly related to this issue, but important to this form.

- Increased max applicants from 3 to 25
- Added a couple unit tests for previously uncovered functionality
- Fixes a missing string formatting visual bug on `missingfileoverview` component
- Updated E2E test data to include > 3 applicants in a test
- Team: IVC CHAMPVA
- Flipper: NA

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/85713
- https://github.com/department-of-veterans-affairs/vets-api/pull/17076

## Testing done

- Manual
- Verified unit and E2E tests pass

## Screenshots

|         | Before | After |
| ------- | ------ | ----- |
| Increased |Screenshot 2024-06-12 at 13 43 09|Screenshot 2024-06-12 at 13 42 06|
|unrelated visual bug|bug2|Screenshot 2024-06-12 at 13 32 03|

## What areas of the site does it impact?

This form only

### Quality Assurance & Testing

- [X] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [X] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [X] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

NA

### CODE REVIEW

File: src/applications/ivc-champva/10-10D/tests/e2e/fixtures/data/test-data.json
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/tests/e2e/fixtures/data/test-data.json:1]: Updated this test data file to include > 3 applicants for E2E tests

File: src/applications/ivc-champva/10-10D/tests/unit/containers/IntroductionPage.unit.spec.js
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/tests/unit/containers/IntroductionPage.unit.spec.js:1]: Unrelated to the main task of increasing applicants - this was not previously covered by a unit test, so verifying it renders to increase coverage

File: src/applications/ivc-champva/10-10D/tests/unit/helpers/helpers.unit.spec.js
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/tests/unit/helpers/helpers.unit.spec.js:1]: Unrelated, but this was not covered by a test previously. Added one to increase coverage.

File: src/applications/ivc-champva/shared/components/fileUploads/MissingFileOverview.jsx
Comment (michaelclement) [inline src/applications/ivc-champva/shared/components/fileUploads/MissingFileOverview.jsx:1]: Fix for the visual bug (see screenshots)

**Review by reiting — APPROVED** [review]

**Review by balexandr — APPROVED** [review]

**Review by CBonade — APPROVED** [review]

**Pull request merged**